gossip/src/cluster_info.rs

```diff
@@ -2845,11 +2845,19 @@ impl Node {
             Self::get_gossip_port(gossip_addr, port_range, bind_ip_addr);
         let (tvu_port, tvu) = Self::bind(bind_ip_addr, port_range);
         let (tvu_forwards_port, tvu_forwards) = Self::bind(bind_ip_addr, port_range);
-        let (tpu_port, tpu) = Self::bind(bind_ip_addr, port_range);
-        let (_tpu_port_quic, tpu_quic) = Self::bind(
-            bind_ip_addr,
-            (tpu_port + QUIC_PORT_OFFSET, tpu_port + QUIC_PORT_OFFSET + 1),
-        );
+        let mut tpu_ports = None;
+        // Try to find 2 consecutive ports
+        for _ in 0..10 {
+            let (tpu_port, tpu) = Self::bind(bind_ip_addr, port_range);
```

Comment:
On each iteration is it guaranteed that the same port won't be bound twice?

Reply:
it's not